pkg/controller/constants.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2020 the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+const (
+	Backup                = "backup"
+	BackupDeletion        = "backup-deletion"
+	BackupStorageLocation = "backup-storage-location"
+	BackupSync            = "backup-sync"
+	DownloadRequest       = "download-request"
+	GarbageCollection     = "garbage-collection"
```

Comment:
Sorry, I missed this in the first pass :( This will also change the value for `--disable-controllers` like the `restic-repo` value. It's currently `gc` and will be changed to `garbage-collection`

Reply:
argh! Thank you